frameworks/hdfs/tools/docker-client/configure-hdfs.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+usage() {
+    echo "HDFS_SERVICE_NAME=hdfs /$HADOOP_VERSION/configure-hdfs.sh"
+    exit 1
+}
+
+[[ -n "${HDFS_SERVICE_NAME}" ]] || usage
+
+cd /$HADOOP_VERSION
```

Comment:
It's out of scope of this PR, but it's on my mind since we are considering the same thing in TensorFlow. Would it not make sense to rather accept the full URL to the where `core-site.xml` and `hdfs-site.xml` could be found.

This is the approach used by Spark (https://docs.mesosphere.com/services/spark/2.1.0-2.2.1-1/hdfs/) and it would be useful to have consistency.